streams/src/test/java/org/apache/kafka/streams/integration/ProcessingExceptionHandlerIntegrationTest.java

```diff
@@ -222,9 +221,9 @@ public void shouldStopOnFailedProcessorWhenProcessingExceptionOccursInContinuePr
     }
 
     @Test
-    public void shouldStopProcessingWhenFatalUserExceptionInFailProcessingExceptionHandler() {
+    public void shouldStopProcessingWhenProcessingExceptionHandlerReturnsNull() {
```

Comment:
Re-purposed this test case for NULL case.

It seem redundant to test "throw exception" twice as originally in the code (one for FAIL and once for CONTINUE), because if we throw, there is no response we return at all...